doc/source/programs/gdal_raster_pixelinfo.rst

```diff
@@ -0,0 +1,238 @@
+
+    Query the (overview_level)th overview (overview_level=0 is the 1st overview),
+    instead of the base band. Note that the x,y location (if the coordinate system is
+    pixel/line) must still be given with respect to the base band.
+
+Examples
+--------
+
+.. example::
+   :title: Reporting on pixel column=5, line=10 on the file :file:`byte.tif`
+
+   .. code-block:: console
+
+       gdal raster pixelinfo byte.tif 5 10
+
+   outputs:
+
+   .. code-block:: json
+
+       {
+          "type":"FeatureCollection",
+          "crs":{
+            "type":"name",
+            "properties":{
+              "name":"urn:ogc:def:crs:EPSG::26711"
+            }
+          },
+          "features":[
+            {
+              "type":"Feature",
+              "properties":{
+                "input_coordinate":[
+                  5.0,
+                  10.0
+                ],
+                "column_int":5,
+                "line_int":10,
+                "column_double":5.0,
+                "line_double":10.0,
+                "bands":[
+                  {
+                    "band_number":1,
+                    "pixel_value":132
+                  }
+                ]
+              },
+              "geometry":{
+                "type":"Point",
+                "coordinates":[
+                  441020.0,
+                  3750720.0
+                ]
+              }
+            }
+          ]
+        }
+
+
+.. example::
+   :title: Reporting on point at UTM 11N coordinates easting=441320 and northing=3750720 on the file :file:`byte.tif`
+
+   .. code-block:: console
+
+       gdal raster pixelinfo --position-type=geoloc byte.tif 441320 3750720
+
+   outputs:
+
+   .. code-block:: json
+
+       {
+          "type":"FeatureCollection",
+          "crs":{
+            "type":"name",
+            "properties":{
+              "name":"urn:ogc:def:crs:EPSG::26711"
+            }
+          },
+          "features":[
+            {
+              "type":"Feature",
+              "properties":{
+                "input_coordinate":[
+                  441320.0,
+                  3750720.0
+                ],
+                "column_int":10,
+                "line_int":10,
+                "column_double":10.0,
+                "line_double":10.0,
+                "bands":[
+                  {
+                    "band_number":1,
+                    "pixel_value":115
+                  }
+                ]
+              },
+              "geometry":{
+                "type":"Point",
+                "coordinates":[
+                  441320.0,
+                  3750720.0
+                ]
+              }
+            }
+          ]
+        }
+
+
+.. example::
+   :title: Reporting on point at WGS84 coordinates longitude=-117.6355 and latitude=33.8970 on the file :file:`byte.tif`, with CSV output format
+
+   .. code-block:: console
+
+       gdal raster pixelinfo --of=csv --position-crs=WGS84 byte.tif -117.6355 33.8970
```

Comment:
Shouldn't there be also `--position-type geoloc` or `--position-type user-crs` in this example? The default is said to be `pixel`.

Reply:
done